juspay/hyperswitch
# fix(drainer): properly log deserialization errors

## Type of Change


## Description

Added a metric and an error log in case when deserialization fails in Drainer.

## How did you test it?

- Run Older version of hyperswitch-server  and Create a KV enabled Merchant.
- Run a newer version of drainer.
- Create a payment. You can see the deserialization error in the logs

<img width="1728" alt="Screenshot 2023-12-07 at 12 33 15 AM" src="https://github.com/juspay/hyperswitch/assets/43412619/cab457fd-bedd-41eb-97e1-43c9e482ce67">


## Checklist
